docs/rebase.md

```diff
@@ -24,102 +26,100 @@ On the machine used for the rebase,
 
 ### Downloading the target OpenShift release
 
-Run the following to download the OpenShift release to rebase to, specifying the target release image, e.g.:
+Run the following to download the OpenShift release to rebase to, specifying the target release images for _both_ Intel and Arm architectures, e.g.:
 
-```
-./scripts/rebase.sh download registry.ci.openshift.org/ocp/release:4.11.0-0.nightly-2022-06-06-201913
+```shell
+./scripts/rebase.sh download quay.io/openshift-release-dev/ocp-release:4.10.25-x86_64 quay.io/openshift-release-dev/ocp-release:4.10.25-aarch64
 ```
 
-This will create a directory `_output/staging`, download the specified release image's metadata (`release.txt`) and manifests, git-clone (only) the repos of the embedded components and the operators of the loaded components, and check out the commit used by that OpenShift release.
+This will create a directory `_output/staging`, download the specified release images' metadata (`release_{amd64,arm64}.json`) and manifests, git-clone (only) the repos of the embedded components and the operators of the loaded components, and check out the commit used by that OpenShift release.
 
 ### Rebasing the go.mod file and vendoring
 
 In MicroShift's `go.mod` file, we only explicitly add the `require` directives needed by MicroShift itself (marked with the comment `// microshift`) whereas we let `go mod tidy` figure out the direct and indirect requirements for the embedded components. For the rebase, the focus is therefore on the `replace` directives.
 
-When resolving version mismatches between modules used by different embedded components, the general heuristic is to start from a minimal subset of ReplaceDirectives from `o/k` (only those `go mod tidy` actually tries to find) to ensure these are consistent, then add etcd's, openshift-apiserver's and openshift-controller-manager's dependencies.
+When resolving version mismatches between modules used by different embedded components, the general heuristic is to start from a minimal subset of ReplaceDirectives from `o/k` (only those `go mod tidy` actually tries to find) to ensure these are consistent, then add etcd's and openshift-controller-manager's dependencies.
```

Comment:
you were re-adding this in https://github.com/openshift/microshift/pull/808 but I understand it's transient so we can probably ignore this comment.